packages/admin-ui/src/polyfills.ts

```diff
@@ -60,7 +60,7 @@
 import "zone.js";  // Included with Angular CLI.
 import "@angular/localize/init";
 
-
+global.Buffer = global.Buffer || require('buffer').Buffer;
```

Comment:
Why is this now required (not saying it's wrong, just trying to understand the reason we need this from now on)?